app/screens/channel/channel.js

```diff
@@ -167,14 +156,13 @@ class Channel extends PureComponent {
     };
 
     retryLoadChannels = () => {
-        this.loadChannels(this.props.currentTeam.id);
+        this.loadChannels(this.props.currentTeamId);
```

Comment:
I also think we can improve the function for `this.loadChannels` but I could make another PR for that one